app/public/css/styles.css

```diff
@@ -1811,7 +1824,7 @@ li.main-tab input + label.selected:after {
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 56px 48px;
+  padding: 56px 10px;
```

Comment:
@benma I fixed both issues in this PR https://github.com/benma/adalite/pull/2 could you review and squash into your PR?

Besides these 2 issues on some breakpoints (1024-1204px and 768-806px) where the text of the buttons break onto 2 lines, @xdzurman is this acceptable?

![Screen Shot 2021-11-02 at 11 14 38 AM](https://user-images.githubusercontent.com/546900/139828072-572485e2-f987-4a0b-8805-c6dcbd840e10.png)

Reply:
I applied the changes. Please unresolve this thread if needed.

Reply:
Ideally, the first line should be "Unlock with" and the second line the logo of the wallet as this looks a bit awkward and I'd rather avoid it, if possible.

Also, please put the BitBox02 option as the third option, so far we've been adding wallets to this screen in historical order of addition of the respective integrations (Trezor was historically first, then Ledger) the rationale being that we don't want to change the experience for existing users too drastically

Reply:
Had to rewrite the button styling a bit so that the logo wraps onto a new line but everything still stays nicely centered, using flexbox. See screenshots in https://github.com/benma/adalite/pull/4

Also changed the order of the 3 tabs. Should the BitBox02 also be last in all text, i.e. welcome text?
![Screen Shot 2021-11-03 at 7 24 08 PM](https://user-images.githubusercontent.com/546900/140170275-53bf448c-38bc-499f-8899-975761aadd7b.png)

Reply:
> Should I change it to: "It currently supports Trezor model T, Ledger Nano X, Ledger Nano S and BitBox02.

yes, makes sense

Reply:
changed to "Trezor model T, Ledger Nano X, Ledger Nano S and BitBox02" in
https://github.com/benma/adalite/pull/4/commits/6fec66c54d0686d335cef3f13362898fe9423a36

@benma PTAL and squash